platforms/nodejs/versions/stretch/versionsToBuild.txt

```diff
@@ -18,6 +18,7 @@
 14.17.6
 14.18.3
 14.19.1
+14.20.1
```

Comment:
So since stretch is deprecated, we don't want to release any more versions of it. So we will want to remove the changes from this file, since it causes the stretch binaries to be uploaded to the storage account

Reply:
I see. Just reverted the change on stretch. Thanks!